Extensions/xkit_patches.js

```diff
@@ -1294,6 +1515,16 @@ XKit.tools.getParameterByName = function(name){
 
 			},
 
+			/**
+			 * Specify a control button that can be added to future posts
```

Comment:
I updated this and its later use to be "Create a specification for a control button [...]". The most important idea that I'm trying to get across is that it doesn't instantiate anything or actually create the button (despite its name).